include/views/view.h

```diff
@@ -81,10 +80,20 @@ struct VectorView {
   container_t &get_data();
 
   /*!
-   * @brief Returns the displacement
+   * @brief Returns a reference to the container
+   */
+  value_t *get_pointer();
+
+  /*! get_access_displacement.
+   * @brief get displacement from the origin.
    */
   index_t get_access_displacement();
 
+  /*! set_access_displacement.
+   * @brief sett displacement from the origin.
+   */
+  void set_access_displacement();
```

Comment:
This doesn't take a parameter? It looks like it should take a parameter. If it isn't supposed to, maybe enable/disable_access_displacement?

Reply:
No it does not take parameter. It will add the accessor displacement to the accessor. But it will be applied at the beginning of the matrix. it will add the displacement from by adding disp_ to the ptr_ at the begining of each kernel so that the kernel wont repeat this opetration at every eval call

Reply:
Could this be commented and explained please. It’s not clear from reading the name of the method.

Reply:
added